Junk4Dollars-app/ShowAuctions/Auction.swift

```diff
@@ -0,0 +1,6 @@
+struct Auction {
+    var title: String
+    var description: String
+    var starting_price: Int
+    var ends_at: String
```

Comment:
It's a bit odd having `ends_at` be a `String`. Is there a better type you could choose?

Reply:
just changed this to `Date`. I was having trouble bc I didn't realize I needed `Foundation` imported in order to use `Date`